### DISCUSSION TITLE
Will this work with flatpak package?

### DISCUSSION THREAD

**Discussion by butter-fly04** [Ld-Hagen/fix-opera-linux-ffmpeg-widevine#19] [Q&A]
I'm using Opera's flatpack package form flathub on Fedora.

**Comment by Ld-Hagen** [comment]
No. It works with native version only

**Comment by burnout426** [comment]
Try putting libffmpeg.so in "/var/lib/flatpak/app/com.opera.Opera/x86_64/stable/xxx.../files/opera/lib_extra/". It worked for me the last time I tested.
